tasks/eth/013-fp-upgrade-fjord/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson {
+    using LibString for string;
+
+    DisputeGameFactory dgfProxy;
+
+    // Safe contract for this task.
+    address immutable proxyAdminOwnerSafe = vm.envAddress("OWNER_SAFE");
+    address immutable securityCouncilSafe = vm.envAddress("COUNCIL_SAFE");
+    address immutable foundationUpgradesSafe = vm.envAddress("FOUNDATION_SAFE");
+
+    // New dispute game implementations
+    // See governance proposal for verification - https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236
+    FaultDisputeGame constant faultDisputeGame = FaultDisputeGame(0xf691F8A6d908B58C534B624cF16495b491E633BA);
+    PermissionedDisputeGame constant permissionedDisputeGame = PermissionedDisputeGame(0xc307e93a7C530a184c98EaDe4545a412b857b62f);
+
+    /// @notice Sets up the dgfProxy 
+    function setUp() public {
+        string memory addressesJson;
+        // Read addresses json
+        try vm.readFile(
+            string.concat(vm.projectRoot(), "/lib/superchain-registry/superchain/extra/addresses/mainnet/op.json")
+        ) returns (string memory data) {
+            addressesJson = data;
+        } catch {
+            revert("Failed to read lib/superchain-registry/superchain/extra/addresses/mainnet/op.json");
+        }
+        dgfProxy = DisputeGameFactory(stdJson.readAddress(addressesJson, "$.DisputeGameFactoryProxy"));
```

Comment:
This is reverting for me with `no matching value found at "$.DisputeGameFactoryProxy"`. I suspect we need to upgrade the superchain-registry dependency to the latest version.

We also should make sure to add this task to CI, which would also catch this

Reply:
whoops. I had it updated locally but neglected to add that change.
Yeah, a CI check would be nice. But there are alot of broken scripts in superchain-ops as they relied on older versions of submodules.

Reply:
Unless we configure CI to ignore tasks that have been `EXECUTED`.

Reply:
Currently every task gets manually added to CI, check out the diff here: https://github.com/ethereum-optimism/superchain-ops/pull/240/files

Reply:
In other words, this PR would only add this task to CI, not any of the previously executed tasks